src/getjump/Vk/Wrapper/Photos.php

```diff
@@ -56,18 +62,28 @@ public function saveMessagesPhoto($data)
             ->request('photos.saveMessagesPhoto')->one();
     }
 
-    public function uploadAlbum(array $files = array(), $album = false, $group = false)
+    /**
+     * Uploading photos in album
+     *
+     * @param array $listOfFilePaths array with file paths, which you want to upload to an album
+     * @param bool|int|string $album_id ID of album. Album for uploading files
+     * @param bool|int|string $group_id ID of group. Group which belongs album (optional)
+     */
+    public function uploadAlbum(array $listOfFilePaths = [], $album_id = false, $group_id = false)
     {
-        if (sizeof($files) > 5 || sizeof($files) == 0) {
+        if (sizeof($listOfFilePaths) > 5 || sizeof($listOfFilePaths) == 0) {
             // todo Exception
         }
 
-        $server = $this->getUploadServer($album, $group);
+        $server = $this->getUploadServer($album_id, $group_id);
 
         $request = $this->guzzle->createRequest('POST', $server->upload_url);
+        /** @var PostBody $postBody */
         $postBody = $request->getBody();
-        foreach ($files as $k => $file) {
-            $postBody->addFile(new PostFile('file' . ($k + 1), fopen($file, 'r')));
+        $fileIndex = 1;
```

Comment:
It's kind a, what old code had doing, just because you will put in `$files` array like ['file1', 'file2'], so array index will start from 0 and code will produce the same. I can't imagine another way, so you will use here.